examples/functions/functions_tester/functions/messageHandler.js

```diff
@@ -0,0 +1,10 @@
+exports = function(changeEvent) {
+    const {fullDocument} = changeEvent;
+    
+    // ... Process the document here ...
+    const someResult = "approved";
+    
+    // Update the document with the result
+    const collection = context.services.get("mongodb-atlas").db("triggerExample").collection("messages");
+    return collection.updateOne({ _id: fullDocument._id, needsTriggerResponse: false, result: someResult });
```

Comment:
this returns a `Promise`. consider refactoring function to use `async` and this line to use `await`?

Reply:
I might agree if this were being used as an example (because people might not know they can just use the async keyword on the function), but returning `await foo` is basically the same as returning the promise, so I'm not sure I agree with you here?

Reply:
to clarify my thought, it'd be refactoring this to be: 

```js
// ..rest of code
const updateResult = await collection.updateOne({ _id: fullDocument._id, needsTriggerResponse: false, result: someResult });
return updateResult;
```

Reply:
I can definitely see where you're coming from